azure-iot-device/azure/iot/device/common/pipeline/pipeline_events_base.py

```diff
@@ -49,21 +49,20 @@ class ResponseEvent(PipelineEvent):
         considered success.
     :type status_code: int
     :ivar request_id: The request ID which will eventually be used to match a RequestOperation
-        operation to this event.
-    :type request: str
+      operation to this event.
+    :type request_id: str
     :ivar response_body: The body of the response.
-    :type request_body: str
-
-    :ivar status_code:
-    :type status: int
```

Comment:
I think status_code got accidentally removed from the documentation block

Reply:
its at the top

---
In reply to: [364355441](https://github.com/Azure/azure-iot-sdk-python/pull/432#discussion_r364355441) [](ancestors = 364355441)